projects/kit/components/tiles/tiles.component.ts

```diff
@@ -30,7 +30,7 @@ import {BehaviorSubject, debounce, filter, map, Subject, timer} from 'rxjs';
     ],
     host: {
         '[class._dragged]': 'element',
-        '(pointerleave.silent)': 'rearrange($event)',
+        '(pointerleave.silent)': 'rearrange(element)',
```

Comment:
There should be nothing:
![image](https://github.com/user-attachments/assets/dc90c33c-a328-48dc-8c77-9a93aa55c436)

Reply:
fixed